Auth0Tests/ResponseSpec.swift

```diff
@@ -38,7 +38,7 @@ class ResponseSpec: QuickSpec {
 
             it("should handle HTTP 204 responses") {
                 let response = Response<AuthenticationError>(data: nil, response: http(204), error: nil)
-                expect(try? response.result()).toNot(beNil())
+                expect(try! response.result()).to(beNil())
```

Comment:
This one is a bit stranger and same for below, following the code the result should be `Nil` although there appears to be a difference in expectation between Swift 4.2 and 5.0 so forced try to ensure Nil vs <Nil>